### PR TITLE
Finished Viscomm Recruit Training

### PR DESCRIPTION
**Pull request by Ahmed-I-Abdullah**

### CODE REVIEW

File: Recruit-Training/Viscomm-Recruit-Training/src/BatteryStateDisplayService.cpp
Comment (DatLam25) [inline Recruit-Training/Viscomm-Recruit-Training/src/BatteryStateDisplayService.cpp:32]: Time should be printed out as HH:MM:SS.ZZZ.

File: Recruit-Training/Viscomm-Recruit-Training/src/BatteryStateOfChargeService.cpp
Comment (DatLam25) [inline Recruit-Training/Viscomm-Recruit-Training/src/BatteryStateOfChargeService.cpp:32]: All instances of `3600000.0` should be replaced with a constant so we can know that the number means.

**Review by Hesham-E — COMMENTED** [review]
Hey this looks great to me! You've got the hang of this so that's awesome. The only thing that I want to mention is that we follow a style guide when programming so that our code is structrued uniformly throughout. For example, we format our if statements like this

if (condition)
{
    code to run
}

This link will explain what you need to know. https://github.com/UCSolarCarTeam/Development-Resources/tree/master/Development-Guide/Style-Guide

**Comment by DatLam25** [comment]
This looks good to me. Well done on finishing the training. Wait for another review before closing this PR.

**Review by DatLam25 — APPROVED** [review]

**Review by Hesham-E — CHANGES_REQUESTED** [review]
Looks good! (Whoops I meant to do this as an approve)

**Review by Hesham-E — APPROVED** [review]
There we go

**Pull request closed without merging**
